nature.bbx

```diff
@@ -18,26 +18,39 @@
 \DeclareBibliographyOption{articletitle}[true]{%
   \settoggle{bbx:articletitle}{#1}%
 }
+\newtoggle{bbx:intitle}
+\DeclareBibliographyOption{intitle}[false]{%
```

Comment:
Sorry to bother you.
Normally all bool options have `\DeclareBibliographyOption{intitle}[true]` so that `\ExecuteBibliographyOptions{intitle}` without an explicit value toggles to true and not false.
Note also that `\DeclareBibliographyOption` now has another optional first argument that specifies the option type: `\DeclareBibliographyOption[boolean]{intitle}[false]`, but I'm not sure if that is backwards-compatible with older versions where the first optional argument wasn't there.